ath-container.sh

```diff
@@ -34,6 +34,7 @@ docker run \
 	--user ath-user \
 	--workdir /home/ath-user/sources \
 	--shm-size 2g \
+	--group-add $(getent group docker | cut -d: -f3) \
```

Comment:
not sure if this will work on mac - does docker-desktop create a group with this name like it does on linux?

Reply:
When dealing with Docker Desktop, we want to retrieve the group on server side, eg. on the VM, not on the client side. You can see that as if your client reaches the remote Docker Engine through SSH.

I have the following command in my archives which I've used for years for this trick (e.g. "to collect the GID of the group owner of the socket in the context of DonD"):

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock alpine stat -c %g /var/run/docker.sock
```

=> works on macOS (Docker Desktop), Linux (both Docker CE and Docker Desktop), Windows (Docker Desktop in Linux container mode) but also any SSH remote Docker

```
# On Ubuntu with Docker CE 27.2.0 - Docker package
$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock ubuntu:noble stat -c %g /var/run/docker.sock      
999
```

```
# On macOS with Docker Desktop latest
$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock ubuntu:noble stat -c %g /var/run/docker.sock      
0
# because it is a special socket shared with the VM  through macOS isolation
```

Reply:
c29029d3dd91d8d06d3487488414a5610c3dccf3